test/v2/connectors/V2BaseConnectorSpec.scala

```diff
@@ -132,6 +132,13 @@ class V2BaseConnectorSpec
       urlPath.toString() mustBe s"/transit-movements-converter/messages/${messageType.code}"
   }
 
+  "the post departure movements URL on localhost for a given departureId should be as expected" in {
+    val departureId = arbitrary[DepartureId].sample.value
+    val urlPath     = Harness.movementsPostDeparture(departureId)
+
+    urlPath.toString() mustBe s"/transit-movements/traders/movements/${departureId.value}/messages/"
+  }
+
```

Comment:
strictly that endpoint is for any movement, not just departure movements. (At least it will be in the future). Not sure if we *need* to rename it, but just an observation.

Reply:
Keeping it is as of now. Will change later if required